client/package.json

```diff
@@ -127,9 +134,12 @@
     "leaflet-gesture-handling": "1.1.8",
     "leaflet.markercluster": "1.4.1",
     "locale-compare-polyfill": "0.0.2",
+    "mathjs": "^6.6.1",
     "lodash": "4.17.19",
     "milsymbol": "2.0.0",
     "moment": "2.27.0",
+    "pathfinding": "^0.4.18",
+    "paths-js": "^0.4.10",
```

Comment:
Pin version number (`yarn add -E pathfinding path-js`) as that is what _renovate_ likes.